src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/SFA.DAS.CommitmentsV2.UnitTests.csproj

```diff
@@ -25,4 +25,13 @@
     <ProjectReference Include="..\SFA.DAS.CommitmentsV2\SFA.DAS.CommitmentsV2.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="SFA.DAS.Apprenticeships.Api.Client">
+      <HintPath>..\..\..\..\das-apprenticeship-programs-api\src\SFA.DAS.Apprenticeships.Api.Client\bin\Debug\netstandard2.0\SFA.DAS.Apprenticeships.Api.Client.dll</HintPath>
+    </Reference>
+    <Reference Include="SFA.DAS.Apprenticeships.Api.Types">
+      <HintPath>..\..\..\..\das-apprenticeship-programs-api\src\SFA.DAS.Apprenticeships.Api.Client\bin\Debug\netstandard2.0\SFA.DAS.Apprenticeships.Api.Types.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
```

Comment:
Does a project reference not work in this case?

Reply:
Yeah, it works (this is a project reference). I guess the hint path was added because the project is not in the same folder may be.

Reply:
Shouldn't it be a nuget reference, and won't it fail on the build server as it won;t be able to find the \das-apprenticeship-programs-api project

Reply:
Ah, I see what you meant. Fixed.